improver/blending/calculate_weights_and_blend.py

```diff
@@ -195,6 +196,7 @@ def process(self, cubelist, cycletime=None, model_id_attr=None,
             result = cube.copy()
             if attributes_dict is not None:
                 amend_attributes(result, attributes_dict)
+            result, = rebadge_forecasts_as_latest_cycle([result], cycletime)
```

Comment:
Should the docstring above the `if` statement be updated, given that this is actually also catching the case where only one cycle has been provided for cycle blending, and the `forecast_reference_time` is being updated, in addition to updating the attributes?

Reply:
Done.